src/mapblock_mesh.cpp

```diff
@@ -914,10 +907,7 @@ static void updateFastFaceRow(
 			if (next_makes_face == makes_face
 					&& next_p_corrected == p_corrected + translate_dir
 					&& next_face_dir_corrected == face_dir_corrected
-					&& next_lights[0] == lights[0]
-					&& next_lights[1] == lights[1]
-					&& next_lights[2] == lights[2]
-					&& next_lights[3] == lights[3]
+					&& memcmp(next_lights, lights, ARRLEN(lights) * sizeof(u16)) == 0
```

Comment:
i hope yes, but as it it's sure it's done correctly, i don't know if compiler algorithm detect we are comparing the same memory area ?